extensions/standard-processors/processors/TailFile.cpp

```diff
@@ -120,56 +158,42 @@ void TailFile::onSchedule(const std::shared_ptr<core::ProcessContext> &context,
   std::string value;
 
   if (context->getProperty(Delimiter.getName(), value)) {
-    delimiter_ = value;
+    delimiter_ = parseDelimiter(value);
+  }
+
+  if (!context->getProperty(FileName.getName(), file_to_tail_)) {
+    throw minifi::Exception(ExceptionType::PROCESSOR_EXCEPTION, "File to Tail is a required property");
```

Comment:
I would suggest using propertybuilder and make the property required, so this can't happen.

Reply:
this is old code just moved around, but good point, I'll change it

Reply:
done